packages/concerto-util/types/lib/basefileexception.d.ts

```diff
@@ -14,8 +14,9 @@ declare class BaseFileException extends BaseException {
      * @param {string} fullMessage - the optional full message text
      * @param {string} [fileName] - the file name
      * @param {string} [component] - the component which throws this error
+     * @param {string} errorType - The optional error code regarding the error
```

Comment:
I'm guessing this is generated?

Reply:
No, I have written that. Should it be removed or should I add `@param {string} [errorType='BaseFileException'] - the error code` ?

Reply:
Could you generate the types then please?  `npx lerna run build`

Reply:
Sorry I think I have resolved the problem.
Generated all the required types.